### DISCUSSION TITLE
report errors-imageryProvider.readyPromise.then(...).otherwise is not a function

### DISCUSSION THREAD

**Discussion by hehexxxgo** [zouyaoji/vue-cesium#181] [Q&A] [answered]
![2022-04-03_22-27-58](https://user-images.githubusercontent.com/94695071/161765301-37358c79-2523-418d-a7d5-dbf2f7e27edd.png)

**Comment by hehexxxgo** [comment]
No problem before, suddenly reported an error

**Comment by j524275214** [comment]
![image](https://user-images.githubusercontent.com/40748734/161912521-7cec12cf-e015-4487-ad10-10da62baee57.png)
you need to find this file in node_modules
and
![image](https://user-images.githubusercontent.com/40748734/161912283-c2a0fba7-7230-44df-95d4-c8c022159b0f.png)

**Comment by hehexxxgo** [comment]
Thank you for your advice and concern!
My problem has been solved.
I didn't specify the version.

**Comment by zouyaoji** [comment, marked as answer]
Please install version 2.3.5
